src/v0/util/facebookUtils/networkHandler.js

```diff
@@ -237,13 +259,19 @@ const errorResponseHandler = (destResponse) => {
     return;
   }
   const { error } = response;
-  const { status, errorMessage, tags: errorStatTags } = getStatus(error);
+  const { status, errorMessage, stats: errorStatTags } = getStatus(error);
+  if (
+    isDefinedAndNotNull(errorStatTags) &&
+    errorStatTags?.[TAG_NAMES.ERROR_TYPE] === 'accessTokenExpired'
```

Comment:
`'accessTokenExpired'` can we use the constant available in `integrations-lib` itself ?

Reply:
changed to TAG_NAMES.ERROR_TYPE.AUTH